pkg/ingest/store/gcs.go

```diff
@@ -51,7 +54,7 @@ func (w *gcsWalker) Walk(ctx context.Context, storageURI *url.URL, op WalkOption
 		}
 		if err := walkFn(ObjectStoreEntry{
 			FullKey:     attrs.Name,
-			RelativeKey: strings.TrimPrefix(attrs.Name, prefix),
+			RelativeKey: strings.TrimPrefix(attrs.Name, basePath),
```

Comment:
Are we okay with this being empty?

Reply:
empty string does nothing to the original value